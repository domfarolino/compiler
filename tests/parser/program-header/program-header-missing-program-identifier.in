program is
begin
end program.
