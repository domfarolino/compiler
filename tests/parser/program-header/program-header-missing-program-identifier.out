[1;31mLine 1: Expected program identifier in program header, but got: 'is'
Line 1: Error parsing program header
Line 1: Expected 'begin' in program body, but got: 'is'
Line 1: Error parsing program body
[0m